scripts/git/clang-format.ps1

```diff
@@ -34,22 +34,22 @@ try {
     }
 
     # Process a single file if specified
-    if ($File) {
+    if ($File -ne "True") {
```

Comment:
I'm sure there's some better way than this to check that $File is empty but they didn't seem to work, would appreciate input from powershell gurus